src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothStructs.cs

```diff
@@ -388,6 +399,21 @@ internal static SocketConnection ConvertStructToSocketConnection(SocketConnectio
             connectionInfo.Uuid = structInfo.ServiceUuid;
             return connectionInfo;
         }
+
+        internal static BluetoothHidDeviceReceivedData ConvertStructToBluetoothHidDeviceReceivedData(BluetoothHidDeviceReceivedDataStruct structInfo)
+        {
+            BluetoothHidDeviceReceivedData receivedData = new BluetoothHidDeviceReceivedData();
```

Comment:
I recommand to make factory method in BluetoothHidDeviceReceivedData

``` c#
  class BluetoothHidDeviceReceivedData {
   ....
   internal static BluetoothHidDeviceReceivedData Create(BluetoothHidDeviceReceivedDataStruct structInfo) {
           BluetoothHidDeviceReceivedData receivedData = new BluetoothHidDeviceReceivedData();
            receivedData.Address = structInfo.RemoteAddress;
            receivedData.HeaderType = structInfo.headerType;
            receivedData.ParamType = structInfo.paramType;
            if (structInfo.dataSize > 0)
            {
                receivedData.DataLength = structInfo.dataSize;
                receivedData.Data = new byte[receivedData.DataLength];
                Marshal.Copy(structInfo.data, receivedData.Data, 0, receivedData.DataLength);
            }
            return receivedData;
   }

  }
```

Reply:
Done.